### PR TITLE
REG-123 Get feature values from pre-calculated bigWig files

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/regulome_search.py [hunk 1]
```diff
@@ -607,12 +611,16 @@ def region_search(context, request):
...
+            evidence = atlas.regulome_evidence(all_hits['datasets'], chromosome, int(start), int(end))
             regdb_score = atlas.regulome_score(all_hits['datasets'], evidence)
+            features = evidence.copy()
+            features.update({
+                k: k in evidence
+                for k in [
+                    'ChIP', 'DNase', 'PWM', 'Footprint', 'eQTL', 'dsQTL',
+                    'PWM_matched', 'Footprint_matched'
+                ]
+            })
```
Comment (hitz) [inline src/encoded/regulome_search.py]: This whole bit seems a little clunky.    Why are we copying the evidence Dict then adding back 8 properties?  Seems like it should be one or the other.    Also this is in there twice, should be factored out for the two views.

File: src/encoded/regulome_atlas.py [hunk 2]
```diff
@@ -256,7 +273,7 @@ def _regulome_category(self, score_category=None, dataset=None):
             return 'Single_Nucleotides'
         return '???'
 
-    def regulome_evidence(self, datasets):
+    def regulome_evidence(self, datasets, chrom, start, end):
```
Comment (hitz) [inline src/encoded/regulome_atlas.py]: Should the name of this function stil be regulome_evidence if it's also getting the scores from the bigWig?

Comment (yunhailuo) [reply src/encoded/regulome_atlas.py]: Scores are part of the features (evidence?) being inputted to the regulome scoring algorithm.

File: src/encoded/regulome_search.py (hunk 1 above)
Comment (yunhailuo) [reply src/encoded/regulome_search.py]: `features` is the exact input into the new scoring algorithm. So I removed details and converted values of those binary variables to boolean. Good to have a function to convert `evidence` to `features` @hitz ?

**Pull request merged**